app/tests/integration/nimbus/test_firefox_targeting.py

```diff
@@ -74,7 +74,7 @@ def targeting_config_slug(request):
 
 
 @pytest.mark.run_targeting
```

Comment:
do you think we should rename this to `run_advance_targeting`? 🤔  just to signify it contains more than required fields

Reply:
Oh this marker is just for selecting tests when pytest runs. These tests should run with the old targeting tests.

Reply:
yup absolutely, I mean to say we are changing the meaning of marker from `targeting` to `advance targeting`, so would recommend refactoring that

Reply:
@yashika-khurana Yes I agree, however I think we should do a larger refactor of all the integration tests to more clearly separate and identify different testing concerns now that we've expanded the scope so much, so I'll file a followup for that with ideas for a better layout.